solr/modules/bert/build.gradle

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'BERT integration package'
+
+dependencies {
+  implementation project(':solr:core')
+  implementation project(':solr:solrj')
+
+  implementation 'org.apache.lucene:lucene-core'
+
+  // implementation 'org.slf4j:slf4j-api'
+
+  testImplementation project(':solr:test-framework')
+  testImplementation 'org.apache.lucene:lucene-test-framework'
+  testImplementation 'com.carrotsearch.randomizedtesting:randomizedtesting-runner'
+  testImplementation 'junit:junit'
+  testImplementation 'org.hamcrest:hamcrest'
+
+  implementation 'com.robrua.nlp:easy-bert'
```

Comment:
so I haven't looked at this much - but @jzonthemtn recently demoed some OpenNLP work that can load bert models I think. I think it was anything from huggingface? I'm not 100% sure. I wonder if that would be potentially an easier integration. OpenNLP I think is already a dependency for analysis or other parts of Solr.

mostly just a comment of "maybe" without any real push for "use this"

Reply:
> ... @jzonthemtn recently demoed some OpenNLP work ...

Perhaps you were referring to https://opensourceconnections.com/blog/2022/06/27/using-modern-nlp-models-from-apache-opennlp-with-solr/ and https://www.youtube.com/watch?v=x5za13Jc5OY which I just came across via the JIRA ticket for #1999 -- interesting stuff, thanks for sharing!